projects/gsc/experiments/sampled_kwinners/sampled_le_sparse_net.py

```diff
@@ -0,0 +1,287 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
```

Comment:
Same as before, was this included intentionally?

Reply:
Removed the changes from the other files.